Neos.Flow/Migrations/Mysql/Version20200908155620.php

```diff
@@ -0,0 +1,75 @@
+<?php
+namespace Neos\Flow\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
```

Comment:
```suggestion
use Doctrine\Migrations\AbstractMigration
```
![image](https://user-images.githubusercontent.com/307571/92720355-c542f400-f364-11ea-9cc8-a06c4980e427.png)

Reply:
Done.